Source/WebCore/PAL/pal/PALSwift/CryptoKitShim.swift

```diff
@@ -476,6 +476,116 @@ public enum EdKeyAgreementAlgorithm {
 }
 
 public class EdKey {
+    public static func generatePrivateKey(algo: EdSigningAlgorithm) -> VectorUInt8 {
+        switch algo {
+        case .ed25519:
+            return Curve25519.Signing.PrivateKey().rawRepresentation.copyToVectorUInt8()
+        case .ed448:
+            return Data(count: 0).copyToVectorUInt8()
+        }
+    }
+
+    public static func generatePrivateKeyKeyAgreement(algo: EdKeyAgreementAlgorithm) -> VectorUInt8
```

Comment:
Nit:  Typo:  "...KeyKey..."?

If this a concatenation of "PrivateKey" and "KeyAgreement", you can ignore this comment.

Reply:
That is what it is `PrivateKey` + `KeyAgreement`